src/main/java/uk/gov/laa/ccms/caab/controller/application/summary/OpponentsSectionController.java

```diff
@@ -275,4 +554,49 @@ private void populateConfirmSharedOrganisationDropdowns(final Model model) {
     model.addAttribute("relationshipsToClient", combinedLookup.getT2().getContent());
   }
 
+  private void populateOrganisationCreateDropdowns(final Model model) {
+    Tuple4<CommonLookupDetail,
+            RelationshipToCaseLookupDetail,
+            CommonLookupDetail,
+        CommonLookupDetail> combinedLookup =
+          Optional.ofNullable(Mono.zip(
+                  lookupService.getCommonValues(COMMON_VALUE_ORGANISATION_TYPES),
+                  lookupService.getOrganisationToCaseRelationships(),
+                  lookupService.getCommonValues(COMMON_VALUE_RELATIONSHIP_TO_CLIENT),
+                  lookupService.getCountries()).block())
+            .orElseThrow(() -> new CaabApplicationException("Failed to retrieve lookup data"));
+
+    model.addAttribute("organisationTypes", combinedLookup.getT1().getContent());
+    model.addAttribute("relationshipsToCase", combinedLookup.getT2().getContent());
+    model.addAttribute("relationshipsToClient", combinedLookup.getT3().getContent());
+    model.addAttribute("countries", combinedLookup.getT4().getContent());
+
+    List<Pair<Boolean, String>> currentlyTradingOptions = List.of(
+        Pair.of(Boolean.FALSE, "No"),
+        Pair.of(Boolean.TRUE, "Yes"));
+    model.addAttribute("currentlyTradingOptions", currentlyTradingOptions);
+  }
+
+  private void populateIndividualCreateDropdowns(final Model model) {
```

Comment:
if common lookups can we use the dropdownBuilder?

Reply:
done